### PR TITLE
Add System config for Supplementary billing flags

### PR DESCRIPTION
**Pull request by Beckyrose200**
https://eaflood.atlassian.net/browse/WATER-4622

As part of the work for two-part tariff supplementary billing, we need to add the system config for the service repo to hit a new endpoint we have created. The endpoint is to flag a licence for supplementary billing. As there are 5 different ways a licence can get flagged with 3 of them triggered from the service repo, this PR will allow us to hit the system endpoint and add the flag if necessary.

### CODE REVIEW

File: src/lib/connectors/system/supplementary-billing.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const config = require('../../../../config')
+const { serviceRequest } = require('@envage/water-abstraction-helpers')
+
+/**
+ * Posts to
+ * @return {Promise}
+ */
+const flagSupplementaryBilling = (chargeVersionId) => {
+  console.log('Charge Version Id :', chargeVersionId)
```
Comment (Cruikshanks) [inline src/lib/connectors/system/supplementary-billing.js:11]: Need to remove the `console.log()`.

File: src/lib/services/charge-versions.js [hunk 2]
```diff
@@ -104,6 +105,11 @@ const persist = async chargeVersion => {
   )
   persistedChargeVersion.chargeElements = await Promise.all(tasks)
 
+  console.log('Charge Version', persistedChargeVersion)
+  console.log('AHHHH Im in service!!!!')
```
Comment (Cruikshanks) [inline src/lib/services/charge-versions.js:109]: Need to remove the `console.log()`s.

**Review by Cruikshanks — CHANGES_REQUESTED** [review]
The repo also has a `.env.example` file that should be updated with the new config.

File: src/lib/services/charge-versions.js (hunk 2 above)
Comment (Beckyrose200) [reply src/lib/services/charge-versions.js:109]: I didn't realise this was up for review sorry, not finished with it!

**Review by Cruikshanks — APPROVED** [review]

**Pull request merged**